### PR TITLE
pdf performance improved

### PR DESCRIPTION
**Pull request by evidentti**

### CODE REVIEW

File: app/scripts/controllers/meeting.details.ctrl.js [hunk 1]
```diff
@@ -45,6 +45,10 @@ angular.module('dashboard')
         var materialsDropdownOpen = false;
         var isIe = $rootScope.isIe;
 
+        function setBlocMode(mode) {
```
Comment (tkenakka) [inline app/scripts/controllers/meeting.details.ctrl.js]: A small typo here, bloc without a k :)

**Comment by tkenakka** [comment]
LGTM

**Comment by evidentti** [comment]
There is problems  with IE

**Comment by tkenakka** [comment]
LGTM
Check width of proposallist/proposal items when there are no items. On IE11 width is not 100% of parent.

**Pull request merged**